ethcore/src/block.rs

```diff
@@ -284,7 +284,6 @@ impl<'x> OpenBlock<'x> {
 		self.block.header.set_difficulty(*header.difficulty());
 		self.block.header.set_gas_limit(*header.gas_limit());
 		self.block.header.set_timestamp(header.timestamp());
-		self.block.header.set_author(*header.author());
```

Comment:
FYI, I removed this one because `author` is now initialized in the constructor of `OpenBlock`. 

Because the actual author of `Clique` is recovered from signature of the header not the author